.eslintrc

```diff
@@ -6,7 +6,6 @@
     "eqeqeq": 2,
     "guard-for-in": 2,
     "indent": [2, 2, {"SwitchCase": 1}],
-    "keyword-spacing": 2,
```

Comment:
The default behavior seams to be :
- before: true
- after : true

src: http://eslint.org/docs/rules/keyword-spacing

but anyway, 2 is not a valid value for this rule